ipfs/ipfs_test.go

```diff
@@ -0,0 +1,133 @@
+package ipfs
+
+import (
+	"context"
+	"log"
+	"testing"
+
+	status "github.com/sonr-io/core/errors"
+)
+
+func TestTempNode(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	/// --- Part I: Create an IPFS API node
+	node, err := SpawnEphemeral(ctx)
+	if err != nil {
+		t.Error(err)
+	}
+	if node == nil {
+		t.Errorf("SpawnDefault(ctx) resulted in nil result")
+		return
+	}
+
+	/// --- Part II: Upload a file to IPFS
+	data := []byte("Hello World!!!")
+	respUp, err := UploadData(ctx, data, node)
+	if err != nil {
+		t.Errorf("UploadData([]byte, coreAPI) resulted in status %d", respUp.Status)
+		t.Error(err)
+	}
+
+	/// --- Part III: Retrieve a file off of IPFS
+	bootstrapNodes := []string{
```

Comment:
Making these a top level constant would reduce code duplication and clean this function up

Reply:
I agree, that would be a bit cleaner